sky/utils/kubernetes_utils.py

```diff
@@ -20,6 +31,22 @@
     'P': 2**50,
 }
 
+logger = sky_logging.init_logger(__name__)
+
+
+class KubernetesNetworkingMode(enum.Enum):
+    """Enum for the different types of networking modes for accessing
+    jump pods.
+    """
+    NODEPORT = 'nodeport'
```

Comment:
Yes, it's supported. In `kubernetes-ray.yml.j2` we changed the service type to a regular `ClusterIP` instead of `NodePort` since all SSH connections go through a jump ssh pod now. 

The SSH Jump Pod [service can still be NodePort](https://github.com/skypilot-org/skypilot/blob/d8302f0dce4a8f9ee9afadba9c9c7368441ba781/sky/templates/kubernetes-sshjump.yml.j2#L54).